tests/test_cxl_component_virtual_switch.py

```diff
@@ -802,15 +919,15 @@ async def wait_and_test_and_stop(vcs: CxlVirtualSwitch):
             cxl_hpa_base = 0x100000000
             await root_port_device.configure_hdm_decoder_single_device(usp, cxl_hpa_base)
 
-            cxl_devices = usp.get_all_cxl_devices()
+            _cxl_devices = usp.get_all_cxl_devices()
```

Comment:
No need for `_`. Not a private variable/method of a class.

Reply:
`cxl_devices` is actually defined earlier, so renaming is required in order for the clean-up to work properly, so I chose `_cxl_devices`.

Maybe we can rename this to `usp_cxl_devices`.

Reply:
Renamed to `usp_cxl_devices`.